test/integration/log_system.cc

```diff
@@ -782,46 +763,6 @@ TEST_F(LogSystemTest, RecordAndPlayback)
   EXPECT_EQ(32, stateMsg.entities_size());
   EXPECT_NE(batch.end(), ++recordedIter);
 
-  // check rest of recordIter (state message) for poses
```

Comment:
We are removing a good number of checks here. My poor understanding make me the question: why not to keep it and just use the other topic name? Are this checks no longer needed?

Reply:
These checks are no longer needed and no longer valid. These checks were used for the extracted `msgs::Pose_V` from the `dynamic_pose/info` topic, but now since `LogRecord` is no longer recording the topic these checks are no longer valid. I've removed these checks and added new tests to check the pose from the `msgs::SerializedStateMap` instead (which comes from the `changed_state` topic)

Reply:
ack! Thanks.